helpers-json/i18next.js

```diff
@@ -52,6 +52,9 @@ exports.Filter = class I18nextFilter {
     }
 
     async translateResource({ resource, translator }) {
+        if (!resource) {
+            return '';
+        }
```

Comment:
why would we want this? an empty file is an invalid json file so it'd fail to parse.